src/components/Pagination/Pagination.tsx

```diff
@@ -20,19 +34,19 @@ export interface PaginationProps extends Omit<ComponentProps<'div'>, 'onChange'>
   /**
    * Options Array of string
    */
-  options?: string[];
+  options: (PaginationOption | PaginationOptionValue)[];
```

Comment:
non-blocking but should we just force this to be `PaginationOptionValue` instead of the mixing and matching?